### PR TITLE
Add intro to IoT

### PR DESCRIPTION
**Pull request by mamidipalli**
Add intro to IoT

### CODE REVIEW

File: english/iot/intro.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Internet of Things
```
Comment (casey-collab) [inline english/iot/intro.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by mamidipalli** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by mamidipalli** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :melon:

**Pull request merged**